submissions/chiconnect-mobile-money-payout/src/App.jsx

```diff
@@ -0,0 +1,184 @@
+import { useEffect, useState } from 'react'
+
+import chimoneyLogo from './assets/chimoney-logo.svg'
+import { API_KEY, getMomoCodes } from './service/fetchApi'
+
+function App() {
+
+  const [error, setError] = useState('')
+  const [info, setInfo] = useState('')
+  const [momoCodes, setMomoCodes] = useState(null)
+
+  const [paymentData, setPaymentData] = useState({
+    country: '',
+    phoneNumber: '',
+    amount: '',
+    momoCode: ''
+  })
+
+  useEffect(() => {
+    const fetchMomoCodes = async () => {
+      const data = await getMomoCodes()
+      setMomoCodes(data)
+    }
+
+    fetchMomoCodes()
+  }, [])
+
+  const handleFormChange = (event) => {
+    const { name, value } = event.target
+    setPaymentData(prevData => {
+      return {
+        ...prevData,
+        [name]: value
+      }
+    })
+  }
+
+  const handlePayClick = (event) => {
+    if (paymentData.country.length === 0) {
+      setError('Select your momo code')
+      return
+    }
+
+    if (paymentData.phoneNumber.length === 0) {
```

Comment:
consider validating the phone number using a regex

Reply:
Noted